app/build.gradle

```diff
@@ -29,9 +29,9 @@ dependencies {
     compile 'com.android.support:design:25.3.1'
     compile 'com.jakewharton:butterknife:8.7.0'
     compile 'com.google.code.gson:gson:2.8.0'
-//    compile 'com.squareup.retrofit2:retrofit:2.1.0'
-//    compile 'com.squareup.retrofit2:converter-gson:2.1.0'
-//    compile 'com.squareup.picasso:picasso:2.5.2'
+    compile 'com.squareup.retrofit2:retrofit:2.1.0'
```

Comment:
David had already asked you to review the dependencies and delete what you are not using at the moment but now you are adding some back.

Pretty sure that gson, retrofit, retrofit converter gson, picasso, android-networking, junit and others maybe are not being used at the moment